### DISCUSSION TITLE
Is GPU compatible with nVidia RTX3060?

### DISCUSSION THREAD

**Discussion by victorypoint** [PaddlePaddle/PaddleOCR#12996] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：Win11
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：PaddleOCR 2.7.0.3, Paddlepaddle-GPU 2.5.2.post117, CUDA 11.7, cuDNN 8.4.1
- 运行指令/Command Code：command-line: paddleocr --image_dir image.png --lang en --use_gpu true
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

I've followed instructions to a T for install of PaddleOCR GPU as specified in -  https://www.paddlepaddle.org.cn/documentation/docs/en/install/pip/windows-pip_en.html. I get no errors but commandline paddleocr with GPU is several seconds slower than CPU. Could it be my nVidia RTX 3060 is not compatible or perhaps Windows 11? Any help is appreciated.

Python 3.10.11
Paddlepaddle-gpu 2.5.2.post117
CUDA 11.7
cuDNN 8.4.1
command-line: paddleocr --image_dir image.png --lang en --use_gpu true

**Comment by dyning** [comment]
建议可以使用PP-ChatOCRv2里面的部署包（包括OCR的推理）试试，https://aistudio.baidu.com/projectdetail/paddlex/7050167
